client/main-layout.tsx

```diff
@@ -372,11 +372,9 @@ export function MainLayout() {
       <ConnectedLeftNav />
       <Content>
         <Switch>
-          <ConditionalRoute
-            path='/admin/:rest*'
-            filters={[IsAdminFilter]}
-            component={LoadableAdminPanel}
-          />
+          <Route path='/admin/:rest*'>
```

Comment:
You can just not render this Route in that case, rather than doing a redirect (not rendering the routes and being on an admin route will make it hit the fallback route here, which will redirect anyway)